pandas/tests/arithmetic/conftest.py

```diff
@@ -90,9 +86,10 @@ def zero(request):
 
 @pytest.fixture(
     params=[
-        Float64Index(np.arange(5, dtype="float64")),
-        Int64Index(np.arange(5, dtype="int64")),
-        UInt64Index(np.arange(5, dtype="uint64")),
+        # TODO: add more  dtypes here
```

Comment:
I've got some problems with arithmetics with non-64-bit indexes. I'd like to take them in a follow-up.